components/drops/view/part/DropPartMarkdown.tsx

```diff
@@ -343,4 +380,4 @@ function DropPartMarkdown({
   );
 }
 
-export default memo(DropPartMarkdown);
+export default memo(DropPartMarkdown);
```

Comment:
```suggestion
export default memo(DropPartMarkdown);

```

Reply:
make sure files have a single blank line at the end. If we are consistent with that everywhere (eg: in editor settings) then it won't show as an extra change here.